src/libraries/System.Private.CoreLib/src/System/Globalization/DateTimeFormat.cs

```diff
@@ -1349,6 +1350,7 @@ internal static unsafe bool TryFormatTimeOnlyR<TChar>(int hour, int minute, int
             }
 
             charsWritten = 8;
+            value.ToDateTime().GetTime(out int hour, out int minute, out int second);
```

Comment:
>value.ToDateTime().GetTime(out int hour, out int minute, out int second); [](http://example.com/codeflow?start=12&length=73)

reducing the parameters count and doing that will reduce the code size? I am trying to understand why doing that.

Reply:
It's reducing parameter count and also doing the hour/minute/second calculation more efficiently since it doesn't calculate them each separately.